### PR TITLE
Added lempransa pond (orsha) as starting map and added team name to query on character creation

### PR DESCRIPTION
**Pull request by huenato**

### CODE REVIEW

**Comment by exectails** [comment]
Looks like you forgot to push the addition of the new db class to the project, resulting in a compilation error.

**Comment by huenato** [comment]
Oops, there you go

File: src/LoginServer/LoginServer.cs [hunk 1]
```diff
@@ -54,7 +54,11 @@ public override void Run()
 			this.CheckDatabaseUpdates();
 
 			// Data
-			this.LoadData(DataToLoad.All, true);
+			this.LoadData(DataToLoad.Jobs, true);
```
Comment (exectails) [inline src/LoginServer/LoginServer.cs]: You could also do this: `this.LoadData(DataToLoad.Jobs | DataToLoad.Maps | DataToLoad.Barracks | [...], true);`

File: system/db/startingcities.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Melia
+// Database file
+//---------------------------------------------------------------------------
+
+[
+// Klaipeda - West Siauliai Woods
+//---------------------------------------------------------------------------
+{ startingCityId: 1, Map: "f_siauliai_west", X: -628, Y: 260, Z: -1025 },
```
Comment (exectails) [inline system/db/startingcities.txt]: Our naming convention for db field names is camelCase, as in the other dbs.

File: src/Shared/Data/Database/StartingCities.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+using System;
+using Newtonsoft.Json.Linq;
+using Melia.Shared.Const;
+
+namespace Melia.Shared.Data.Database
+{
+	[Serializable]
+	public class StartingCityData
+	{
+		public int Id { get; set; }
```
Comment (exectails) [inline src/Shared/Data/Database/StartingCities.cs]: Why not use the StartingCity enum for the id and the index?

**Comment by huenato** [comment]
I'll update the code in the next 2 days. I'll be a bit busy for a while.

**Pull request merged**